src/Pos/Explorer/DB.hs

```diff
@@ -28,6 +30,17 @@ import           Pos.Txp.Toil          (Utxo)
 import           Pos.Util.Chrono       (NewestFirst (..))
 
 ----------------------------------------------------------------------------
+-- Types
+----------------------------------------------------------------------------
+
+type Page = Integer
+type Epoch = Integer
```

Comment:
> I don't like to wrap and unwrap code too much, the client is sending an Integer,

Then transformation should be done quickly after receiving it from the client. I don't know that part well, but virtually all `cardano-sl` uses `EpochIndex` to refer to epoch. And block stores it as `EpochIndex`. So it doesn't make sense to use something different, it will most likely involve even more wrapping and unwrapping eventually and it's inconsistent.
If epoch arrives as `Integer` from the client, it's ok, but it should be converted to appropriate type as soon as possible.
Also `Integer` doesn't make sense here, because it doesn't seem possible that epoch will ever be greater than 10¹⁸.